data/autoyast_sle15/pc_tools.xml

```diff
@@ -99,7 +99,6 @@
             <package>ntp</package>
             <package>gcc</package>
             <package>python3-pip</package>
-            <package>aws-cli</package>
```

Comment:
actually initial idea is to move everything into autoyast and fully drop usage of prepare_tools module , so you getting fully functional image after applying autoyast . But after your PR I realized that I overlook that in prepare_tools we installing awscli with pip so this is probably actual root cause of https://progress.opensuse.org/issues/54602. @jlausuch  ^^^^